api/v1alpha1/openapi.yaml

```diff
@@ -1523,6 +1525,57 @@ paths:
               schema:
                 $ref: '#/components/schemas/Status'
 
+  /api/v1/deviceactions/resume:
+    post:
+      tags:
+        - deviceactions
+      description: Resume devices based on label selector and/or field selector.
+      operationId: resumeDevices
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DeviceResumeRequest'
+        required: true
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: integer
+                description: Number of devices that were successfully resumed
+              example: 3
```

Comment:
Make it an object so that it's extensible

Reply:
fixed